anime/Services/ImportProgramService.php

```diff
@@ -24,7 +24,7 @@
 //     'comment'     Description of the event. May be NULL.
 //     'hidden'      Whether the event should be publicly visible.
 //     'locationId'  Internal id of the location in the AnimeCon database.
-//     'floor'       Floor on which the event takes place. Prefixed with 'floor-'. {-1, 0, 1, 2}.
+//     'floor'       Floor on which the event takes place. Prefixed with 'floor-'. <digit>.
```

Comment:
Technically <digit> would only encapsulate [0-9] :-). I'm OK with the generalization so let's merge this, I was just lazy since it's rather unlikely that the World Forum will magically gain new floors.